bin/compile

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
Better to use #!/bin/sh since some one might not have bash

Reply:
This file is running in Heroku space and users doesn't have to run this file manually. Then we can keep this as it is.